terraform/modules/s3-bucket/main.tf

```diff
@@ -0,0 +1,15 @@
+resource "aws_s3_bucket" "s3_bucket" {
+  bucket = "${var.bucket_name}"
+  acl    = "private"
+  policy = <<EOF
+{
+  "Version": "2012-10-17",
+  "Statement": [{
+    "Effect": "Allow",
+    "Principal": {"AWS": "arn:aws:iam::${var.access_account_id}:root"},
+    "Action": "*",
```

Comment:
Why are the `List` and `Get` actions required on the bucket? What would a user not be able to do without them?